src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+syntax = "proto3";
+
+package aspire.v1;
+
+import "google/protobuf/timestamp.proto";
+
+////////////////////////////////////////////
```

Comment:
I expect these to go over this channel too. Initially we'll implement the protocol here in-proc. It may need further extension before we can cross the process/network boundary.

Reply:
I pushed an update that includes a first draft of logs and telemetry.

Reply:
Thanks!

Reply:
Sounds like console logs are in scope, but telemetry will happen via standard OTLP mechanisms, so I'll remove the telemetry stuff I added.